audit/src/test/java/gov/cms/ab2d/audit/cleanup/FileDeletionServiceTest.java

```diff
@@ -63,62 +60,66 @@ private void changeFileCreationDate(Path path) throws IOException {
 
     @Test
     public void checkToEnsureFilesDeleted() throws IOException, URISyntaxException {
+        String efsMount = tmpDirFolder.toPath().toString();
+
         // other tests set this value, so set it to the correct one, JUnit ordering annotations don't seem to be respected
         ReflectionTestUtils.setField(fileDeletionService, "efsMount", efsMount);
 
         // Don't change the creation date on this file, but do so on the next ones
         Path destinationNotDeleted = Paths.get(efsMount, TEST_FILE_NOT_DELETED);
         URL urlNotDeletedFile = this.getClass().getResource("/" + TEST_FILE_NOT_DELETED);
         Path sourceNotDeleted = Paths.get(urlNotDeletedFile.toURI());
-        Files.move(sourceNotDeleted, destinationNotDeleted, StandardCopyOption.REPLACE_EXISTING);
+        Files.copy(sourceNotDeleted, destinationNotDeleted, StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Curious what caused the change from `move` to `copy`? CC @adaykin .

Reply:
One thing I found strange was that when I was executing this test file from IntelliJ, it was deleting the file from `src/test/resources` so the next run it will fail saying `file not found`.
Had a chat with @adaykin  yesterday and his suggestion was to try copy instead of move and see if it works. I was able to get it running, so went with that solution.

Reply:
I believe there were issues running the test in the IDE where the file was being removed with the move.

Reply:
👍